app/gui/src/integration_test.rs

```diff
@@ -0,0 +1,155 @@
+//! Enso testing fixtures
```

Comment:
1. missing dot
2. The first paragraph should explain what this module is about, but it does not. However, the second paragraph does a way better job at that. Maybe we should just remove the first paragraph and leave the second only? 
3. I know this file was refactored and not created in this PR, but lets fix it if its changed.